### DISCUSSION TITLE
How to add remove button like this button in the node.

### DISCUSSION THREAD

**Discussion by Azim-Ahmed** [xyflow/xyflow#1524] [Q&A]
![image](https://user-images.githubusercontent.com/67516192/134241697-72e5482d-0b8c-4499-a0ee-13a6083787dd.png)

**Comment by FredrikAugust** [comment]
You can use custom nodes which can render any React component, and from there it should be straight forward. Consult the documentation for custom nodes/examples.

**Comment by brenolapreza** [comment]
Hi,

Did you manage to solve this problem? If yes, could you share? Tks!

**Reply by FredrikAugust** [reply]
See my comment. Use a custom `nodeType` and just render your own component, in which you can put whatever your heart desired.
